src/attachments/modifyImageProcess.js

```diff
@@ -35,4 +37,16 @@ if (imageWidth && imageHeight) {
       console.log("Could not reduce image", reduceOptions, err);
       process.exit(1);
     });
+} else {
+  const trimOptions = {
+    fromPath: fromPath,
+    toPath: toPath
+  };
+  imageResizer
+    .trimImage(trimOptions)
+    .then(() => process.exit(0))
+    .catch(err => {
+      console.log("Could not trim image", trimOptions, err);
+      process.exit(1);
+    });
```

Comment:
Hier ist glaube ich irgendwas durcheinander gekommen.

Die console logs passen nicht zum else Zweig (trim und reduce vertauscht) und im letztendlichen else müsste doch wieder minify enthalten sein oder?

Reply:
Hm, sehe keinen Fehler.
* Im 1. Zweig sind ja `imageWidth`und `height` gesetzt und damit muss ein resize gemacht werden. Minify und trim gehen gleichzeitig und werden als Optionen durchgereicht.
* Im 2. Zweig haben wir keinen resize aber minify ist true. Also muss das `reduceImage` ausgeführt werden. Trim ist gleichzeitig möglich, also wird es als Option mitgegeben.
* Im 3. Zweig (also dem else) sind wir nur, wenn weder resize noch minify gemacht werden sollen. Bleibt nur noch trim übrig. Darum wird direkt das `trimImage` aufgerufen. Das trimmed und macht sonst nix.

Reply:
Passt in meinen Augen auch.